exercise46.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+FILENAME = 'words.txt'
+string_of_words = ''
+
+with open(FILENAME) as f_object:
+    for contents in f_object:
+        normalized_content = contents.lower().strip("\n")
+        string_of_words += normalized_content + ' '
+
+list_of_words = string_of_words.split()
```

Comment:
I tried testing your regex but couldn't replicate it. Maybe we can sit down together...

It _should_ work as `[^a-zA-Z0-9_]` is the definition of `\W`